dan_layer/engine_types/src/confidential/claim.rs

```diff
@@ -3,12 +3,14 @@
 
 use serde::{Deserialize, Serialize};
 use tari_bor::{borsh, Encode};
+use tari_common_types::types::PublicKey;
 use tari_crypto::ristretto::RistrettoComSig;
-use tari_template_lib::models::LayerOneCommitmentAddress;
+use tari_template_lib::models::UnclaimedConfidentialOutputAddress;
 
 #[derive(Debug, Clone, Encode, Deserialize, Serialize, Eq, PartialEq)]
 pub struct ConfidentialClaim {
-    pub commitment_address: LayerOneCommitmentAddress,
+    pub diffie_hellman_public_key: PublicKey,
```

Comment:
I prefer `public_key` and to allow use cases where a different scheme is used. It might always be DH, but someone might come up with a different approach and then the name is mismatched

Reply:
yeah that makes sense, wasn't sure about that name lol